Solutions/Corvus.Tenancy.Storage.Azure.Blob/Corvus/Tenancy/Storage/Azure/TenantProviderBlobStore.cs

```diff
@@ -189,6 +203,19 @@ public async Task<ITenant> CreateChildTenantAsync(string parentTenantId, string
             {
                 throw new TenantNotFoundException();
             }
+            catch (StorageException ex) when (ex.RequestInformation.HttpStatusCode == (int)HttpStatusCode.Conflict)
+            {
+                // This exception is thrown because there's already a tenant with the same Id. This should never happen; when
```

Comment:
"This should never happen". I think that's true if you're not using well-known ids, but it's not entirely unexpected if we are using well-known ids. (It's still fairly unlikely - would only happen if we have a network failure where we can't tell whether our attempt to create the well-known thing succeeded, and we try again, only to find out that the previous request did in fact succeed.) Consider dropping the "This should never..." sentence, and also "However" from the start of the next sentence